docs/maatregelen/kwetsbare_groepen.md

```diff
@@ -0,0 +1,57 @@
+vereiste:
+- tbd
+levenscyclus:
+- ontwerp
+bouwblok:
+- fundamentele-rechten
+rollen:
+- projectleider
+- opdrachtgever
+- ethicus
+hide:
+- navigation
+---
+
+<!-- tags -->
+
+## Maatregel
+Bepaal wat de impact van het in te zetten algoritme is voor betrokkenen (personen of groepen).
+Bepaal vervolgens of er groepen zijn waarbij de impact van het algoritme dermate groot kan zijn, dat het wenselijk is om deze groepen extra bescherming te bieden.
+
+## Toelichting
+- Verschillende groepen kunnen op een andere manier geraakt worden door het inzetten van een algoritme. Dit is afhankelijk van de context waarin het algoritme wordt ingezet, en dient daardoor bij iedere toepassing opnieuw bekeken te worden. 
+- Bedenk wat er met de uitkomsten van het algoritme gedaan wordt, en wat de consequenties daarvan zijn voor burgers. Hierbij kan gedacht worden aan de volgende aspecten:
+    - Worden bepaalde groepen sneller gemonitord?
+    - Wat als het model het fout heeft? 
+    - Wordt het systeem gebruikt om informatie te verkrijgen, om besluiten voor te bereiden of om zelfstandige besluiten te nemen en welke gevolgen heeft dat voor de mate waarin het algoritme bepalend zal zijn in de praktijk? 
+    - Worden de gegevens veilig en vertrouwelijk behandeld; welke gevolgen zou een datalek hebben voor groepen of categorieën personen?
+    - Worden data gedeeld met andere partijen en wat is het gevaar dat die misbruik maken van de data met negatieve gevolgen voor groepen of categorieën personen?
+- Houd hierbij ook rekening met de impact van het in te zetten algoritme op de samenleving (vanuit sociaal, democratisch en milieu/ecologisch perspectief).
+- Om de impact op groepen te bepalen, kan het handig zijn een mensenrechtentoets zoals het [Impact Assessment Mensenrechten en Algoritmes](https://open.overheid.nl/documenten/ronl-c3d7fe94-9c62-493f-b858-f56b5e246a94/pdf) toe te passen. 
+- Bepaal of er maatregelen genomen kunnen worden om de geïdentificeerde groepen extra bescherming te bieden. Hierbij kan men denken aan de volgende aspecten: Kan de (extra) administratieve druk voor bepaalde groepen worden weggenomen? Worden resultaten van het algoritme naast de resultaten van een expert gelegd? Is het wenselijk om een proces in te richten waarbij zowel algoritme als een expert een uitkomst geven? Kunnen we de betreffende groep extra hulp aanbieden? Is het wenselijk bij negatieve uitkomsten een vier-ogen-principe toe te passen? 
+- De impact van het algoritme op de groepen die geïdentificeerd worden in deze stap, kunnen mogelijk onderzocht worden in een [biasanalyse](../bouwblokken/fundamentele-rechten/non-discriminatie/). <!-- verwijzing toevoegen --> Daarbij kan geidentificeerd worden of bepaalde groepen oververtegenwoordigd of ondervertegenwoordigd zijn in selecties, of dat het algoritme andere of meer fouten maakt voor bepaalde groepen. 
```

Comment:
heir moest nog een verwijzing worden toegevoegd zag ik. Heb nu de link naar de algemene pagina van Bias en Non-discriminatie toegevoegd, is dat okay of bedoelde je iets anders?

Reply:
ik bedoelde hier eigenlijk naar de maatregel over het uitvoeren van een biasanalyse (die maatregel bestaat nu nog niet), maar denken we in de werkgroep wel al over na. Voor nu prima zo denk ik :)